src/lib/containers/CardContainer.tsx

```diff
@@ -82,12 +78,8 @@ export const CardContainer = (props: CardContainerProps) => {
       </div>
     )
   } else if (data && data.queryResult!.queryResults.rows.length === 0) {
-    // data was retrieved from the backend but there is none to show.
-    if (queryRequest.query.additionalFilters) {
-      return <SearchResultsNotFound />
-    }
-    // else show "no results" UI (see PORTALS-1497)
-    return <NoContentAvailable />
+    // Show "no results" UI (see PORTALS-1497)
+    return <NoContentPlaceholder />
```

Comment:
`additionalFilters` will be used for cases other than column search, so this no longer makes sense (and prompted this entire PR)